metadata/src/main/java/com/redhat/lightblue/metadata/ValueGenerator.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.lightblue.metadata;
+
+import java.io.Serializable;
+import java.util.Properties;
+
+/**
+ * Represents value generator definition located in entityInfo.
+ *
+ * @author mpatercz
+ *
+ */
+public class ValueGenerator implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    public enum ValueGeneratorType {
+        IntSequance;
+    }
+
```

Comment:
You mean value generator definition in entityInfo is to be handled by value generator type to/from json methods?
```
           {
                "name": "testSeq",
                "type": "IntSequance",
                "configuration": [
                    {
                        "name": "min",
                        "value": "10000"
                    },
                    {
                        "name": "max",
                        "value": "99999"
                    }
                ]
            }
```